packages/api/src/hooks/useRemoteConfig.ts

```diff
@@ -0,0 +1,23 @@
+import { useQuery } from '@tanstack/react-query';
+import initData from '../remote_config.json';
+
+const remoteConfigQuery = async function () {
+    if (!process.env.REMOTE_CONFIG_URL) {
+        throw new Error('Remote Config URL is not set!');
+    }
+    const response = await fetch(process.env.REMOTE_CONFIG_URL);
+    if (!response.ok) {
```

Comment:
is this an Deriv API, or external API? 
(I do not see the URL so I cannot validate).

If its an external API, then I guess security needs to have its say

Reply:
its an external endpoint
https://deriv-static.firebaseio.com/remote_config/deriv-app.json

Reply:
moving forward we will proxy pass it using cloudflare

Reply:
I'm assuming this was verified and accepted as generally use of external api always carries risk and whole bunch of consequences (access management, security implications, who can and who cannot access it etc)

Reply:
The Dashboard and the Platform that we are going to use eventually has not been confirmed yet. But there should be no problem changing it if we just keep the naming conventions and the schema the same as I proposed here.